osu.Server.Spectator/Hubs/Multiplayer/Matchmaking/Queue/MatchmakingQueueBackgroundService.cs

```diff
@@ -185,6 +187,8 @@ public async Task ExecuteOnceAsync()
                 logger.LogError(ex, "Failed to refresh the matchmaking queue.");
             }
 
+            await refreshPools();
```

Comment:
Should probably try-catch this so that the entire background service doesn't fall over if anything in this dies.